<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LongLatCalcDialogBase</class>
 <widget class="QDialog" name="LongLatCalcDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>119</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CÃ¡lculo Longitude e Latidude GMS - G</string>
  </property>
  <widget class="QLabel" name="lblLong">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Longitude</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spbLongD">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>21</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="suffix">
    <string> D</string>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="value">
    <number>45</number>
   </property>
  </widget>
  <widget class="QLabel" name="lblLat">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Latitude</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spbLatD">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="suffix">
    <string> D</string>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="maximum">
    <number>90</number>
   </property>
   <property name="value">
    <number>23</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spbLongM">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="suffix">
    <string> M</string>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="maximum">
    <number>59</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="displayIntegerBase">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spbLatM">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="suffix">
    <string> M</string>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="maximum">
    <number>59</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="displayIntegerBase">
    <number>10</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spbLongS">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="suffix">
    <string> S</string>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="maximum">
    <double>59.990000000000002</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spbLatS">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="suffix">
    <string> S</string>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="maximum">
    <double>59.990000000000002</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbLong">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>20</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>W</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>E</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="cmbLat">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>70</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>S</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>N</string>
    </property>
   </item>
  </widget>
  <widget class="QDoubleSpinBox" name="spbLatDD">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>70</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="suffix">
    <string> DD</string>
   </property>
   <property name="decimals">
    <number>6</number>
   </property>
   <property name="minimum">
    <double>-90.000000000000000</double>
   </property>
   <property name="maximum">
    <double>90.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>-23.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spbLongDD">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>20</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="suffix">
    <string> DD</string>
   </property>
   <property name="decimals">
    <number>6</number>
   </property>
   <property name="minimum">
    <double>-180.000000000000000</double>
   </property>
   <property name="maximum">
    <double>180.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>-45.000000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
